contracts/ILockManager.sol

```diff
@@ -2,5 +2,6 @@ pragma solidity ^0.4.24;
 
 
 interface ILockManager {
+    function lockCallback(uint256 _amount, uint256 _allowance, bytes _data) external returns (bool);
```

Comment:
What do you think about `receiveLockManager` or similar? I'd add the address of the user as well